pmm-app/tests/DbaaS/api/dbaasAPI.js

```diff
@@ -40,6 +40,18 @@ module.exports = {
     );
   },
 
+  async apiDeletePSMDBCluster(dbClusterName, clusterName) {
+    const body = JSON.stringify({ kubernetes_cluster_name: `${clusterName}`, name: dbClusterName });
```

Comment:
```suggestion
    const body = JSON.stringify({ kubernetes_cluster_name: clusterName, name: dbClusterName });
```
Can we do like this?

Reply:
ofcourse, will try

Reply:
Done